tests/cli/core/accounts/api/test_account_api_service.py

```
@@ -5,15 +5,15 @@
 from mpt_api_client.exceptions import MPTHttpError
 
 
-def test_get_authentication_success(mock_mpt_api_client, mock_mpt_api_tokens):
+def test_get_authentication_success(api_mpt_client, mock_mpt_api_tokens):
     secret = "idt:TKN-1111-1111:secret"
```

Ruff (0.15.2)

[error] Possible hardcoded password assigned to: "secret"

(S105)